lib/teleterm/clusters/cluster_auth.go

```diff
@@ -186,6 +186,11 @@ func (c *Cluster) login(ctx context.Context, sshLoginFunc client.SSHLoginFunc) e
 		return trace.Wrap(err)
 	}
 
+	// Attempt device login. This activates a fresh key if successful.
+	if err := c.clusterClient.AttemptDeviceLogin(ctx, key); err != nil {
```

Comment:
FYI @ravicious, this wires device login into Connect. I don't expect problems, but just a heads up.